demos/linear-progress.html

```diff
@@ -16,7 +16,6 @@
     <meta charset="utf-8">
     <title>Linear Progress - Material Compoonents Catalog</title>
     <meta name="viewport" content="width=device-width, initial-scale=1">
-    <script src="assets/material-components-web.css.js" charset="utf-8"></script>
```

Comment:
Removing this currently breaks this demo's styles because the next line is broken - `demos.css` should be `assets/demo-styles.css.js`